test/services/roster/index.test.js

```diff
@@ -49,6 +50,23 @@ describe('roster service', function oauth() {
 			key: '1',
 			oAuthClientId: '123456789',
 		};
+		testToolTemplate = await toolService.create(testToolTemplate);
+		testTool1 = {
+			_id: '5a79cb15c3874f9aea14daa5',
+			name: 'test1',
+			url: 'https://tool.com?pseudonym={PSEUDONYM}',
```

Comment:
Please don't use URLs who whe don't have control about it.
https://tool.com is a validate URL with a webseit behind it.